lib/main.dart

```diff
@@ -85,6 +88,9 @@ class ChatScreenState extends State<ChatScreen> with TickerProviderStateMixin {
       };
       _messagesReference.push().set(message);
     });
+    setState(() {
+      _isComposing = false;
```

Comment:
is this necessary?

Reply:
Yep, this ensures the submit button is deactivated on iOS. It seems _handleMessageChanged is not called here.
(tested with the iOS simulator iPhoneX)